tests/node/peer_manager/peer_store/test_migrations.nim

```diff
@@ -0,0 +1,56 @@
+import std/[options], stew/results, testutils/unittests
+
+import
+  ../../../../waku/[node/peer_manager/peer_store/migrations],
+  ../../waku_archive/archive_utils,
+  ../../testlib/[simple_mock]
+
+import std/[tables, strutils, os], stew/results, chronicles
+
+import ../../../../waku/[common/databases/db_sqlite, common/databases/common]
+
+suite "Migrations":
+  test "migrate ok":
+    # Given the db_sqlite.migrate function returns ok
+    let backup = db_sqlite.migrate
+    mock(db_sqlite.migrate):
+      proc mockedMigrate(
+          db: SqliteDatabase, targetVersion: int64, migrationsScriptsDir: string
+      ): DatabaseResult[void] =
+        ok()
+
+      mockedMigrate
+
+    # When we call the migrate function
+    let migrationResult = migrations.migrate(newSqliteDatabase(), 1)
+
+    # Then we expect the result to be an error
```

Comment:
Comment is wrong I think, not an error. Right?

Reply:
Yes, fixed.